### DISCUSSION TITLE
How to get Github studentpack

### DISCUSSION THREAD

**Discussion by CodeLK4** [community/community#65090] [GitHub Education]
### Hi everyone,

How to get Github studentpack

**Comment by LinukaAr** [comment]
Create a GitHub Account:
If you don't have a GitHub account, you'll need to create one. Visit the GitHub website and sign up for an account using your email address.

Visit the Student Developer Pack Page:
Go to the GitHub Education website and find the Student Developer Pack page. You can usually find this by searching for "GitHub Student Developer Pack" on a search engine.

Start the Application Process:
On the Student Developer Pack page, there should be a button or link that allows you to start the application process. Click on that button.

Complete the Application:
You'll be asked to provide information about yourself and your educational institution. You might need to provide your school email address for verification purposes. Follow the instructions and fill out the required details.

Verify Your Student Status:
Depending on the verification method your institution uses, you might need to provide documents or information to prove your student status. This could include uploading a scan of your student ID, providing your school email address, or using an identity verification service.

Wait for Approval:
After submitting your application and verification documents, you'll need to wait for GitHub's approval. This process might take a few days.

Receive Access:
Once your application is approved, you'll receive access to the GitHub Student Developer Pack. This might include free access to tools like GitHub Pro, JetBrains IDEs, domain registration, hosting services, and more.

Please note that the application process and requirements might change over time. It's recommended to visit the official GitHub Education website for the most up-to-date information on how to apply for the Student Developer Pack.

Keep in mind that my information might be outda

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
